src/sentry/charts/metric_alerts.py

```diff
@@ -0,0 +1,223 @@
+from datetime import datetime, timedelta
```

Comment:
I don't know if this belongs in this module, if anything I might keep this in `src/sentry/incidents/charts.py` or something similar.

The `charts` module was designed to be a generic way to produce charts, so keeping code specific to metric alerts in here doesn't seem quite right